### PR TITLE
Change 'Azure' → 'Microsoft Azure'

### PR DESCRIPTION
**Pull request by ahmetalpbalkan**
Also this needs to be updated at https://coreos.com/os/docs/latest/

image

### CODE REVIEW

File: os/booting-on-azure.md [hunk 1]
```diff
@@ -49,10 +49,10 @@ startup, and more via cloud-config. Jump over to the [docs to learn about the
 supported features][cloud-config-docs]. Cloud-config is intended to bring up a
 cluster of machines into a minimal useful state and ideally shouldn't be used
 to configure anything that isn't standard across many hosts. Once an instance
-is provisioned on Azure, the cloud-config cannot be modified.
+is provisioned on Microsoft Azure, the cloud-config cannot be modified.
 
-You can provide raw cloud-config data to CoreOS
-<a href="#via-the-cross-platform-cli">via the Azure cross-platform CLI</a>.
+You can provide raw cloud-config data to CoreOS <a href="#via-the-cross-
```
Comment (marineam) [inline os/booting-on-azure.md]: I'm guessing the href string shouldn't be line wrapped.

Comment (ahmetalpbalkan) [inline os/booting-on-azure.md]: @marineam ah true, fixed now.

Comment (joshix) [inline os/booting-on-azure.md]: TODO: joshix: [Style guide](https://github.com/coreos/docs/blob/master/STYLE.md) fixes: Unwrap lines. Links should be markdown not raw html.

**Comment by joshix** [comment]
Thanks for the sharpening, @ahmetalpbalkan ! The full "Microsoft Azure" name will be in the next site sync.

**Pull request merged**